packages/debug/src/browser/debug-session-manager.ts

```diff
@@ -287,6 +287,16 @@ export class DebugSessionManager {
         return this.start(options);
     }
 
+    async terminateSessions(): Promise<void> {
+        this.updateCurrentSession(undefined);
+        this.currentSession?.terminate();
+    }
+
+    async restartSessions(): Promise<void> {
+        this.updateCurrentSession(undefined);
+        this.currentSession?.restart();
+    }
```

Comment:
I'm not sure I understand these changes. The `updateCurrentSession` sets the `currentSession` as `undefined`.
This means that `terminate` and `restart` will never occur.

Reply:
```
protected updateCurrentSession(session: DebugSession | undefined): void {
        this.currentSession = session || this.sessions[0];
    }
```
passing `undefined` updates `currentSession` to session[0] - which is what I wanted in order to terminate and restart successfully sessions.

Reply:
Thank you, I had missed the part where if `undefined` it would pick up the next session.